cypress/integration/movie_details_spec.js

```diff
@@ -83,14 +83,14 @@ describe('Movie Details user flows', () => {
   });
 
   it('Should show a 404 Error if an invalid URL is entered', () => {
-    cy.visit('http://localhost:3000/showmemovies')
```

Comment:
really weird that the api returns a 500 instead of a 404 if the /:id has an alphabetic character, but this controls for that. This behavior, and our control of it, was run by our instructors and Kayla confirmed we are handling it the way she would.

Nice test!